014-realtime-api.md

```diff
@@ -90,8 +91,67 @@ Possible protocols to take under future considiration:
 - SSE support: https://github.com/hhxsv5/php-sse
 - Socket.io support: https://github.com/shuixn/socket.io-swoole-server
 
-### Channels and Messages
+### Channels
+
+| Channel                    | Description                                                  |
+| -------------------------- | ------------------------------------------------------------ |
+| account                    | All account related events (session create, name update...)  |
+| collections                | Any update/delete/create events for collections where user has read access |
+| collections.[ID]           | Any update/delete/create events to a given collection where user has read access |
+| collections.[ID].documents | Any update/delete/create events to any document in a given collection where user has read access |
+| documents                  | Any update/delete/create events to documents where user has read access |
+| documents.[ID]             | Any update/delete/create events to a given document where user has read access |
+| files                      | Any update/delete/create events to file where user has read access |
+| files.[ID]                 | Any update/delete/create events to a given file where user has read access |
+| functions                  | Any execution for functions where user has read access       |
+| functions.[ID]             | Any execution event to a given function where user has read access |
+| functions.executions       | Any update to executions where user has read access          |
```

Comment:
In most cases we have been avoiding a prefix to channel names. `functions.executions` => `executions`

Reply:
Agree 👍🏻